### PR TITLE
Make Pokémon Sprite Visualizer name official

### PR DESCRIPTION
**Pull request by Bassoonian**
With #4719 officially naming the Pokémon Debug menu (the one you open with select on the summary screen) the Pokémon Sprite Visualizer (see the readme on upcoming), this PR clarifies the name by renaming all the in-game assets to refer to the Pokémon Sprite Visualizer, allowing for disambiguation with the other debug menus and officially making the name canonical.

## **Discord contact info**
bassoonian

### CODE REVIEW

File: include/constants/pokemon_sprite_visualizer.h [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
-#define DEBUG_MON_SHINY 0
-#define DEBUG_MON_NORMAL 9
-#define DEBUG_FOLLOWER_X 176
-#define DEBUG_FOLLOWER_Y 128
+#define PSV_MON_BACK_X 62
+#define PSV_MON_BACK_Y 80
+#define PSV_ICON_X 224
+#define PSV_ICON_Y 144
+#define PSV_FOLLOWER_X 176
+#define PSV_FOLLOWER_Y 128
```
Comment (AsparagusEduardo) [inline include/constants/pokemon_sprite_visualizer.h:10]: Based on previous "PSS" confusion, maybe we could change the short prefix from `PSV` to `VISUALIZER`

File: include/pokemon_sprite_visualizer.h [hunk 2]
```diff
@@ -55,7 +55,7 @@ struct PokemonDebugMenu
     u8 frontShadowSpriteId;
     bool8 isShiny;
     bool8 isFemale;
-    struct PokemonDebugModifyArrows modifyArrows;
+    struct PokemonSpriteVisualizerModifyArrows modifyArrows;
     struct PokemonDebugOptionArrows optionArrows;
     struct PokemonDebugYPosModifiyArrows yPosModifyArrows;
```
Comment (AsparagusEduardo) [inline include/pokemon_sprite_visualizer.h:60]: Aren't these other structs gonna be renamed?

File: src/pokemon_sprite_visualizer.c [hunk 3]
```diff
@@ -398,16 +398,16 @@ const u8 gBattleBackgroundTerrainNames[][26] =
...
-static void ResetBGs_Debug_Menu(u16);
-static void Handle_Input_Debug_Pokemon(u8);
-static void ReloadPokemonSprites(struct PokemonDebugMenu *data);
-static void Exit_Debug_Pokemon(u8);
+static bool32 TryMoveDigit(struct PokemonSpriteVisualizerModifyArrows *modArrows, bool32 moveUp);
+static void CB2_Pokemon_Sprite_Visualizer_Runner(void);
+static void ResetBGs_Pokemon_Sprite_Visualizer(u16);
+static void Handle_Input_Pokemon_Sprite_Visualizer(u8);
+static void ReloadPokemonSprites(struct PokemonSpriteVisualizer *data);
+static void Exit_Pokemon_Sprite_Visualizer(u8);
```
Comment (AsparagusEduardo) [inline src/pokemon_sprite_visualizer.c:410]: I don't like the underscore usage for these functions originally, so I would recommend something like this:
```suggestion
static void CB2_PokemonSpriteVisualizerRunner(void);
static void ResetBGs_PokemonSpriteVisualizer(u16);
static void HandleInput_PokemonSpriteVisualizer(u8);
static void ReloadPokemonSprites(struct PokemonSpriteVisualizer *data);
static void Exit_PokemonSpriteVisualizer(u8);
```

**Review by AsparagusEduardo — CHANGES_REQUESTED** [review]

**Comment by Bassoonian** [comment]
Feedback integrated

**Review by AsparagusEduardo — APPROVED** [review]

**Pull request merged**